packages/list/index.tsx

```diff
@@ -364,6 +364,19 @@ export default class List extends React.Component<ListProps, ListState> {
     this.setState({listItemClassNames});
   };
 
+
+  getListProps = (checkboxList?: boolean, radioList?: boolean) => ({
```

Comment:
I believe you should place the call to `memoizeOne` directly here. If it's in render, the cache will still be reseted.

(sorry for commenting so much 😅)

Reply:
I think my last comment maybe wasn't clear enough, sorry

Reply:
I believe you're right again sir! 

(please keep the comments coming)